po/ar.po

```diff
@@ -347,129 +347,129 @@ msgstr "تمرين: فيبوناتشي"
 
 #: src/SUMMARY.md src/std-types.md
 msgid "Standard Library Types"
-msgstr ""
+msgstr "أنواع المكتبة القياسية"
 
 #: src/SUMMARY.md src/std-types/std.md
 msgid "Standard Library"
-msgstr ""
+msgstr "المكتبة القياسية"
 
 #: src/SUMMARY.md src/std-types/docs.md
 msgid "Documentation"
-msgstr ""
+msgstr "المستندات التوثيقية (Documentation)"
 
 #: src/SUMMARY.md
 msgid "`Option`"
-msgstr ""
+msgstr "الخيار (Option)"
 
 #: src/SUMMARY.md src/error-handling/result.md
 msgid "`Result`"
-msgstr ""
+msgstr "النتيجة (Result)"
 
 #: src/SUMMARY.md src/android/aidl/types/primitives.md
 #: src/android/interoperability/cpp/type-mapping.md
 #, fuzzy
 msgid "`String`"
-msgstr "السلاسل النص او الكتابة (Strings)"
+msgstr "السلاسل النصية (Strings)"
 
 #: src/SUMMARY.md src/std-types/vec.md
 msgid "`Vec`"
-msgstr ""
+msgstr "المصفوفات (Vectors)"
 
 #: src/SUMMARY.md src/std-types/hashmap.md src/bare-metal/no_std.md
 msgid "`HashMap`"
-msgstr ""
+msgstr "خريطة المشفرة (HashMap)"
 
 #: src/SUMMARY.md src/std-types/exercise.md
 #, fuzzy
 msgid "Exercise: Counter"
-msgstr "تمرين: فيبوناتشي"
+msgstr "تمرين: عداد"
 
 #: src/SUMMARY.md src/std-traits.md
 msgid "Standard Library Traits"
-msgstr ""
+msgstr "سمات المكتبة القياسية"
 
 #: src/SUMMARY.md src/std-traits/comparisons.md
 #: src/concurrency/welcome-async.md
 msgid "Comparisons"
-msgstr ""
+msgstr "مقارنات"
 
 #: src/SUMMARY.md src/std-traits/operators.md
 msgid "Operators"
-msgstr ""
+msgstr "المشغلات (`Operators`)"
```

Comment:
Since backticks are for code, this might be a mistake? In an English text, I would probably write the foreign word in italics, but of course I don't know what the tradition is for Arabic.

Reply:
```suggestion
msgstr "المشغلات (Operators)"
```